bang/models.py

```diff
@@ -692,6 +692,28 @@ def cached_chibis(self):
             'http_image_url': get_http_image_url_from_path(path),
         }) for id, path in zip(split_data(self._cache_chibis_ids), split_data(self._cache_chibis_paths))]
 
+    cameo_members = models.ManyToManyField(Member, related_name='cameo_members', verbose_name=_('Cameos'))
```

Comment:
Can you move it up with the rest of fields? I only list the `_` fields at the end. Under the chibis would make sense.